tests/test_devices.py

```diff
@@ -612,6 +612,41 @@
             "upgradable": False,
             "uplink": PDU_PRO["uplink"],
             "uptime": 1347471,
+            "uptime_stats": {
```

Comment:
Now its added to the test fixture, now you just need to add it to a test as well to increase coverage

Reply:
I'm trying to make it run locally. Having some issues. 
Is there a guide to make it work? 

I run Python 3.9.6

I've run `./setup.sh` 
Should that be it?

Reply:
I'm not sure how strict it will be with python version but I think I'm using python 3.10 or 3.11 specific stuff. Best would be to upgrade to python 3.12

After running the setup script you need to source venv/bin/activate